shared/src/types/vote_extensions/ethereum_events.rs

```diff
@@ -79,11 +79,11 @@ pub struct MultiSignedEthEvent {
     pub event: EthereumEvent,
     /// List of addresses of validators who signed this event
     #[cfg(feature = "abcipp")]
-    pub signers: HashSet<Address>,
+    pub signers: BTreeSet<Address>,
     /// List of addresses of validators who signed this event
     /// and block height at which they signed it
     #[cfg(not(feature = "abcipp"))]
-    pub signers: HashSet<(Address, BlockHeight)>,
+    pub signers: BTreeSet<(Address, BlockHeight)>,
 }
```

Comment:
`MultiSignedEthEvent` and `EthMsgUpdate` are now the same for ABCI++ and may become the same for ABCI+ as well soon. These structs could probably be combined in the future - a `MultiSignedEthEvent` translates straightfowardly into an `EthMsgUpdate`.